### ISSUE TITLE
When aborting a search request the last active symbol should be shown again

### ISSUE THREAD

**Issue by egraether** [CoatiSoftware/CoatiBugTracker#325]

**Comment by mlangkabel** [comment]
Does this still make sense? And what is meant by "abording a search request"? Hitting escape before hitting return?

**Comment by egraether** [comment]
Yes, it makes sense. After removing the last match and typing something, then unfocusing without searching, the field should return to displaying the currently active symbol.
